elem.go

```diff
@@ -91,6 +93,26 @@ func (e *Element) RenderTo(builder *strings.Builder) {
 	builder.WriteString(`>`)
 }
 
+// return string representation of given attribute with its value
+func (e *Element) renderAttr(attrName string) string {
+	var builder strings.Builder
```

Comment:
This creates a new string for every attribute and slows down performance. Can we pass in the builder like `appendAttrToBuilder(builder *strings.Builder, attrName string)` and append the string to the existing builder instead of creating a new one?

Reply:
Thanks for pointing this out, I fixed this to use a builder.